src/sys/mod.rs

```diff
@@ -38,7 +38,13 @@ pub mod mman;
 
 pub mod pthread;
 
-#[cfg(any(target_os = "android", target_os = "linux"))]
+#[cfg(any(target_os = "dragonfly",
```

Comment:
These should be in alphabetical order.

Reply:
Fixed, squashed and pushed. Sorry about that!